backend/src/test/e2e/auth.test.mjs

```
@@ -8,7 +8,7 @@ import chai from "./index.mjs";
 
 const user = mocks.UserBuilder.user;
 
-describe("E2e tests auth", () => {
+describe.only("E2e tests auth", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)